### PR TITLE
Increase Test coverage for IsoParser to 100% and Pep8

### PR DESCRIPTION
**Pull request by alimcmaster1**
## Summary of changes
Increase test coverage of IsoParser to 100% and PEP8

Closes <#521>

### CODE REVIEW

File: dateutil/test/test_isoparser.py [hunk 1]
```diff
@@ -220,50 +249,59 @@ def test_iso_ordinal(isoord, dt_expected):
...
-    ('2012-W01-8', ValueError),                 # Invalid ISO week day
-    ('2013-000', ValueError),                   # Invalid ordinal day
-    ('2013-366', ValueError),                   # Invalid ordinal day
-    ('2013366', ValueError),                    # Invalid ordinal day
-    ('2014-03-12Т12:30:14', ValueError),        # Cyrillic T
-    ('2014-04-21T24:00:01', ValueError),        # Invalid use of 24 for midnight
-    ('2014_W01-1', ValueError),                 # Invalid separator
-    ('2014W01-1', ValueError),                  # Inconsistent use of dashes
-    ('2014-W011', ValueError),                  # Inconsistent use of dashes
+    ('201', ValueError),  # ISO string too short
```
Comment (pganssle) [inline dateutil/test/test_isoparser.py]: I prefer the original comment alignment. PEP 8 does not require them to not be aligned.

File: dateutil/test/test_isoparser.py [hunk 2]
```diff
@@ -274,7 +312,7 @@ def test_iso_raises_sep(sep_act, valid_sep):
 
 @pytest.mark.xfail()
 @pytest.mark.parametrize('isostr,exception', [
-    ('20120425T01:2000', ValueError),           # Inconsistent time separators
+    ('20120425T01:2000', ValueError),  # Inconsistent time separators
```
Comment (pganssle) [inline dateutil/test/test_isoparser.py]: Another change to revert.

File: dateutil/test/test_isoparser.py [hunk 3]
```diff
@@ -283,7 +321,6 @@ def test_iso_raises_failing(isostr, exception):
         isoparse(isostr)
 
 
-###
```
Comment (pganssle) [inline dateutil/test/test_isoparser.py]: Don't delete these "header" comments.

**Comment by pganssle** [comment]
1. I would prefer if large-scale formatting changes were submitted in a separate commit from actual content
2. Most of these changes are not for PEP 8 compliance and actually make this file harder to read. Please don't change the alignment of the comments or the `###` "header" comments.

**Comment by alimcmaster1** [comment]
Sure, thanks for the feedback,  you want me to resubmit with just the test case? If desired I can keep the PEP8 fixes and remove my changes that touched comment spacing and ### headers.

**Comment by pganssle** [comment]
@alimcmaster1 Doesn't have to be resubmitted, but whatever's easiest, just as long as the PEP 8 changes are in a separate commit from the substantive changes. You can rewrite the history and use `git push --force` if you need to.

File: dateutil/test/test_isoparser.py [hunk 4]
```diff
@@ -162,36 +188,40 @@ def test_isoparse_sep_none(datestr, sep):
...
+
 ##
 # Uncommon date formats
 TIME_ARGS = ('time_args',
-    ((None, time(0), None), ) + tuple(('%H:%M:%S.%f', _t, _tz)
-        for _t, _tz in it.product([time(0), time(9, 30), time(14, 47)],
-                                  TZOFFSETS)))
+             ((None, time(0), None),) + tuple(('%H:%M:%S.%f', _t, _tz)
+                                              for _t, _tz in
+                                              it.product([time(0), time(9, 30), time(14, 47)], TZOFFSETS)))
```
Comment (jbrockmendel) [inline dateutil/test/test_isoparser.py]: wont pep8 complain about a long line here (E501 I think)?

File: dateutil/test/test_isoparser.py (hunk 3 above)
Comment (jbrockmendel) [reply dateutil/test/test_isoparser.py]: The pep8-compliant way to do this I've seen elsewhere is `# --------------------------------------`

File: dateutil/test/test_isoparser.py [hunk 5]
```diff
@@ -474,10 +510,10 @@ def test_isotime_raises(isostr, exception):
...
-    ('14:3015', ValueError),                    # Inconsistent separator use
-    ('201202', ValueError)                      # Invalid ISO format
+    ('14:3015', ValueError),  # Inconsistent separator use
+    ('201202', ValueError)  # Invalid ISO format
 ])
 def test_isotime_raises_xfail(isostr, exception):
     iparser = isoparser()
     with pytest.raises(exception):
-        iparser.parse_isotime(isostr)
+        iparser.parse_isotime(isostr)
```
Comment (jbrockmendel) [inline dateutil/test/test_isoparser.py]: pep8 complaint about missing '\n' at end of file

**Comment by alimcmaster1** [comment]
Thanks both for the feedback. Since <#745> seems to be discussing styling in general in the project I will leave my pep8 changes for now. Have revert and just submitted the additional test case I added.

What do you think?

**Comment by alimcmaster1** [comment]
Will raise a new PR with no formatting changes etc

**Pull request closed without merging**